2-collectors/scala-stream-collector/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/model.scala

```diff
@@ -61,6 +61,11 @@ package model {
     enabled: Boolean,
     placeholder: Option[String]
   )
+  final case class RootResponseConfig(
+    statusCode: Option[Int] = None,
+    headers: Option[Map[String, String]] = None,
+    body: Option[String] = None
```

Comment:
I think we can have `404` as default for `statusCode` and `Map.empty[String, String]` for `headers`, wdyt?

Reply:
No issues with that.